pkg/plugin/run.go

```diff
@@ -218,7 +218,8 @@ func (e *AutoscaleEnforcer) handleResources(
 	vCPUTransition := collectResourceTransition(&node.vCPU, &pod.vCPU)
 	memTransition := collectResourceTransition(&node.memSlots, &pod.memSlots)
 
-	vCPUVerdict := vCPUTransition.handleRequested(req.VCPU, startingMigration)
+	vCPUVerdict := vCPUTransition.handleRequested(
+		req.VCPU, startingMigration)
```

Comment:
```suggestion
	vCPUVerdict := vCPUTransition.handleRequested(req.VCPU, startingMigration)
```

unrelated formatting change?